sunpy/map/mapbase.py

```diff
@@ -2708,20 +2708,23 @@ def plot(self, *, annotate=True, axes=None, title=True, autoalign=False,
 
         return ret
 
-    def contour(self, level, **kwargs):
+    def contour(self, level, method='skimage', **kwargs):
```

Comment:
We now need to raise a deprecation warning, saying that skiimage will be dropped in future and we will default to contourpy